src/domain/opportunity/opportunity.resolver.fields.ts

```diff
@@ -44,11 +40,7 @@ export class OpportunityResolverFields {
     return groups;
   }
 
-  @Roles(
-    RestrictedGroupNames.CommunityAdmins,
-    RestrictedGroupNames.EcoverseAdmins,
-    RestrictedGroupNames.Members
-  )
+  @Roles(RestrictedGroupNames.Members)
```

Comment:
This will restrict only to the Members, not all levels. Roles need to be added explicitly.